### ISSUE TITLE
Tweaks not working

### ISSUE THREAD

**Issue by meetdilip** [vinceliuice/Orchis-theme#291]
Hi, 

I installed it by using a simple` ./install.sh` and it worked. But I had a bug with dual monitor in Inkscape. When I try to open  a file, it is half in one monitor and half in another. So I cannot open any file using Inkscape.

Due to this bug, I uninstalled Orchis theme sing ` ./install.sh  -u " and problem starts.

Whatever I do, my theme won't change using Gnome Tweaks. I even tried using the Extension Manager tool. 

The only thing that works is `Settings` dark and light theme switch.

Just now I used `./install -r` and Yaru blue and other accent colours are working. But sadly, when I choose Yaru -dark - blue, it won't switch to dark using Tweaks.

In short, something broke Gnome Tweaks on my Ubuntu 22.04. 

It would be nice if you can help me fix the Tweaks issue. Thanks.

**Comment by vinceliuice** [comment]
Did you installed libadwaita fix?

**Comment by vinceliuice** [comment]
You need run `./install.sh -l -r`

**Comment by meetdilip** [comment]
Thanks for the help. I didn't use` -l ` while installing. Is that causing the issue regarding Inkscape?

I just tried the command you gave. Now the Tweaks work, but only after log out and log  in.

**Comment by vinceliuice** [comment]
This just make dark mode and theme switch not work, I don't think it will cause inkscape issues

**Comment by meetdilip** [comment]
Even now I am having problems using Tweaks. I tried all 3

./install.sh -u

./install.sh -r

./install.sh  -l -r

But unable to switch themes using Tweaks or Extension Manager

I even went through the process of installing and uninstalling again to confirm. Still unable to switch themes. Only Settings > Light / Dark will work :(

**Comment by vinceliuice** [comment]
Libadwaita not support theme switch anymore, it’s not a bug, just gnome team do not want support custom themes anymore

**Comment by meetdilip** [comment]
I removed everything related to Orchis to the best of my effort. Any chance I can get the Tweaks back as it was before trying the install.sh ? Thanks.

**Comment by Demetrio92** [comment]
@meetdilip alt+f2, then `rt` or better `r` after EVERY change. 

Also, go complain to the gnome devs, and be glad there's a libadwaita fix provided at all.

**Comment by meetdilip** [comment]
@Demetrio92 I am not sure I understood the context

I am simply requesting help to go back to a condition before I executed the `./install.sh`

**Comment by vinceliuice** [comment]
If you want system setting and nautilus ...  apps switch theme by gnome-tweaks you need go back to gnome-shell version < = 42.0, It's not the theme cause tweaks not work, it Gnome do not support theme switch after version 42.0 you understand?

**Comment by vinceliuice** [comment]
If you still think it's the theme cause it, you can install a new system in virtualbox to see if it can switch themes by tweaks

**Comment by meetdilip** [comment]
Ok, thanks for the info.

**Comment by meetdilip** [comment]
Tweaks was working for me. Perhaps because I upgraded from 20.04. After trying `install.sh`, I lost the ability to change themes using Tweaks. Thanks for the all the inputs.

**Comment by Demetrio92** [comment]
@meetdilip use do `install.sh --uninstall`. Cuz if you used install with a `--libadwaita` flag it linked some directories in your `$HOME/.config/gtk4.0` 
see: https://github.com/vinceliuice/Orchis-theme/blob/master/core.sh#L199

This probably messes up the other themes. There is no other way to make libadwaita apps to follow themes. If you have a backup for your gtk4.0 folder, restore it, and all will be good.

**Comment by meetdilip** [comment]
Thanks @Demetrio92 , I went for a fresh install this time.
